ui/components/src/utils/chart-actions.ts

```diff
@@ -56,14 +56,17 @@ export function restoreChart(chart: EChartsInstance) {
  * Clear all highlighted series when cursor exits canvas
  * https://echarts.apache.org/en/api.html#action.downplay
  */
-export function clearHighlightedSeries(chart: EChartsInstance, totalSeries: number) {
+export function clearHighlightedSeries(chart: EChartsInstance) {
   if (chart.dispatchAction !== undefined) {
-    for (let i = 0; i < totalSeries; i++) {
```

Comment:
Not directly related to the Firefox issue, but ECharts doesn't actually require you set an explicit seriesIndex this way in dispatchAction. It can automatically "unhighlight" the correct series (which is way better for perf when moving your cursor between quickly between panels)